cumulusci/tasks/marketing_cloud/get_user_info.py

```diff
@@ -0,0 +1,45 @@
+from logging import getLogger
+
+import requests
+
+from cumulusci.utils.http.requests_utils import safe_json_from_response
+
+from .base import BaseMarketingCloudTask
+from .mc_constants import MC_API_VERSION
+
+
+class GetUserInfoTask(BaseMarketingCloudTask):
+    """Retrieves user info of pertaining to the currently logged in user.
+    Sanitizes and returns the payload in self.return_values."""
+
+    logger = getLogger(__name__)
```

Comment:
You don't need or want this; task instances get initialized with a logger that is derived from the flow in some cases: https://github.com/SFDO-Tooling/CumulusCI/blob/main/cumulusci/core/tasks.py#L87